ruby/vampires.rb

```diff
@@ -0,0 +1,55 @@
+Vampire_Names = "dra drac ula cula fang nosf eratu"
+Allergies = %w(peanuts sunshine wheat beestings)
+Employee_Roster = []
+
+
+def vamp_namer(name)
+	vamp_array = Vampire_Names.split(" ")
+	vamp_array.each { |vamp_ele| return true if name.downcase.include?(vamp_ele) }
+	false
+end
+
+puts "How many employees to process?"
+num = gets.chomp.to_i
+
+num.times do
+	employee_allergies = Hash.new(0)
+
+	puts "What is your name?"
+	name = gets.chomp
+
+	puts "How old are you?"
+	age = gets.chomp.to_i
+
+	puts "What year were you born?"
+	birth_year = gets.chomp.to_i
+
+	puts "Our company cafeteria serves garlic bread. Should we order some for you? (y / n)"
+	garlic_order = gets.chomp
+
+	puts "Would you like to enroll in the company’s health insurance? (y / n)"
+	insurance = gets.chomp
+
+	Allergies.each do |allergy|
+		puts "Are you allergic to #{allergy}? (y / n)"
+		response = gets.chomp
+		employee_allergies[allergy] = response
+	end
+
+	case
+	when age < 116 && birth_year > 1900 && !vamp_namer(name)
```

Comment:
Calling a comparison operator returns a boolean value of true or false.  If we want to convert the user input into a boolean value we may be inclined to do something like:
````
if user_input == 'y'
	my_var = true
else
	my_var = false
end
```
which is ok, but if we leverage our knowledge of return values we can set our variable my_var to the return of the comparison like:

```my_var = user_input == ‘y’```

much cleaner! We can then use those boolean values in our conditionals in order to clean up the conditionals and make our code more readable.